### ISSUE TITLE
update webpage /help to reflect new urls for fem, ifem, netfem and mblock manuals

### ISSUE THREAD

**Issue by heyram** [UIUC-PPL/charm#55]
*Original issue: https://charm.cs.illinois.edu/redmine/issues/55*

---
These dormant manuals are now under /www/manuals/old/*

**Comment by heyram** [comment]
*Original date: 2013-02-21 15:47:37*

---
Ramprasad Venkataraman wrote:
> These dormant manuals are now under /www/manuals/old/*

* The charisma manual has also been taken offline (being 4+ years old, and quite out of date). The link needs to be removed from that page.
* The installation and usage manual has been folded into the main charm++ manual. That link needs to be deleted.
* We are no longer building ps versions of any manual. All links to ps files need to be deleted.

**Comment by PhilMiller** [comment]
*Original date: 2013-03-08 02:32:50*

---
Ping on this - some younger student should be shown how to get through all this, and ensure that it's documented.

**Comment by heyram** [comment]
*Original date: 2013-03-08 23:51:52*

---
Meanwhile, we have broken links on our webpage. This should be a quick fix for someone who can update and deploy the PPL website.

Documentation on how to do this should be added to the wiki.

**Comment by PhilMiller** [comment]
*Original date: 2013-05-09 15:35:33*

---
It looks like there are in fact people interested in using ParFUM. Orion forwarded me the following, to which I'd like to respond "It's been fixed"

> Phil, it looks like the FEM manual isn't getting published here anymore:
> http://charm.cs.illinois.edu/manuals/html/fem/manual.html
> 
> Is this intentional, or a side effect of moving to the new illinois.edu server?
> 
> Is anybody working on the FEM framework or ParFUM anymore?
> 
> ---------- Forwarded message ----------
> From: Andreas Hafver <andreas.hafver`fys.uio.no>
> Date: Wed, May 8, 2013 at 1:28 AM
> Subject: ParFUM framework
> To: olawlor`acm.org
> 
> 
> Dear Sir,
> 
> I am a PhD student at the University of Oslo, Norway, and I am currently working on a new discrete element formulation for simulating coupled fracture and fluid transport.
> I came across your ParFUM / FEM framework earlier this year, and I would like to use your grid class because I require the ability to insert/delete individual nodes/elements in parallel.
> 
> For the past weeks the link to the FEM manual on the Parallel Programming Laboratory website has been broken.
> I was wondering if the framework is still actively maintained and publicly available?
> 
> 
> Best regards,
> 
> Andreas Hafver
>

**Comment by PhilMiller** [comment]
*Original date: 2013-06-17 16:10:35*

---
It looks like these broken links are now simply gone. We should point at the existing documentation, for the time being. Ram gave the base URL in the initial report:
http://charm.cs.illinois.edu/manuals/old

**Comment by nikhil-jain** [comment]
*Original date: 2014-01-06 04:28:08*

---
Inactive for 7 month; Should be a quick task, please complete. Ideally delegate to another student, so that the art of web update is known to more.

**Comment by pplimport** [comment]
*Original author: Shanna DeSouza*
*Original date: 2016-06-14 18:57:19*

---
Old links were still in html commented out, they have been updated and uncommented to display. Update has also been tested and deployed.

**Comment by PhilMiller** [comment]
*Original date: 2016-12-22 23:11:23*

---
Confirmed links now show and point at actual documents.
